src/drivers/power_monitor/ina226/ina226.cpp

```diff
@@ -611,12 +525,26 @@ info()
 static void
 ina2262_usage()
 {
-	PX4_INFO("usage: ina226 command [options]");
-	PX4_INFO("options:");
-	PX4_INFO("\t-b --bus i2cbus (%d)", INA226_BUS_DEFAULT);
-	PX4_INFO("\t-a --all");
-	PX4_INFO("command:");
-	PX4_INFO("\tstart|stop|test|info");
+	PRINT_MODULE_DESCRIPTION(
+		R"DESCR_STR(
+### Description
+Driver for the INA226 power monitor.
+
+Multiple instances of this driver can run simultaneously, if each instance has a separate bus OR I2C address.
+
+For example, one instance can run on Bus 2, address 61, and one can run on Bus 2, address 62.
+)DESCR_STR");
+
+	PRINT_MODULE_USAGE_NAME("ina226", "driver");
+	PRINT_MODULE_USAGE_COMMAND_DESCR("start", "Start a new instance of the driver");
+	PRINT_MODULE_USAGE_PARAM_INT('b', INA226_BUS_DEFAULT, 0, NUM_I2C_BUS_OPTIONS - 1, "I2C bus (default: use board-specific bus)", true);
+	PRINT_MODULE_USAGE_PARAM_FLAG('a', "If set, try to start the driver on each availabe I2C bus until a module is found", true);
+	PRINT_MODULE_USAGE_PARAM_INT('d', INA226_BASEADDR, 0, UINT8_MAX, "I2C Address (in base-10)", true);
+	PRINT_MODULE_USAGE_PARAM_INT('t', 1, 1, 2, "Which battery calibration values should be used (1 or 2)", true);
```

Comment:
Do we only support 2 batteries?

Reply:
Right now in #12551 only 2 batteries are supported. Adding support for additional batteries takes up flash space, which is pretty tight. But if we need to support multiple batteries later, it should be a simple change.